godot-core/src/builtin/string/node_path.rs

```diff
@@ -54,13 +54,14 @@ impl NodePath {
     /// godot_print!("{}", path.get_name(2)); // "Sprite"
     /// ```
     ///
-    /// # Panics
-    /// In Debug mode, if `index` is out of bounds. In Release, a Godot error is generated and the result is unspecified (but safe).
+    /// # Panics (safeguards-balanced)
+    /// If `index` is out of bounds. In safeguards-disengaged mode, a Godot error is generated and the result is unspecified (but safe).
```

Comment:
Not really related to this PR but what does "safe" mean in this context :thinking:?

Reply:
Might be, although "safe" implies "sound".

- Safe: there can be no UB.
- Unsafe: it's the user's responsibility to not cause UB.
- Unsound: there is UB, something needs to be fixed.